### PR TITLE
Library refactoring

### PR DESCRIPTION
**Pull request by akalongman**
- Reformat code to PSR-2 coding standard
- Update Guzzle to 6.x version
- Increase minimum php requirement to 5.5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,83 +46,81 @@ Adding/updating people and scheduling surveys:
...
-$person1 = \Delighted\Person::create(array('email' =>
-                                           'ellie@icloud.com'));
+$person1 = \Delighted\Person::create(['email' => 'ellie@icloud.com']);
 
 // Add a new person, and schedule a survey after 1 minute (60 seconds)
-$person2 = \Delighted\Person::create(array('email' =>
+$person2 = \Delighted\Person::create(['email' =>
                                            'richard.nguyen@aol.com',
-                                           'delay' => 60));
+                                           'delay' => 60]);
```
Comment (e1himself) [inline README.md]: Please fix alignment here and everywhere through README. It looks ugly now :)

File: composer.json [hunk 2]
```diff
@@ -19,8 +15,13 @@
...
+        "php": ">=5.5",
+        "guzzlehttp/guzzle": "^6.2"
+    },
     "require-dev": {
-        "phpunit/phpunit": "~4.5"
+        "phpunit/phpunit": "~4.5",
+        "symfony/var-dumper": "^3.2"
     },
     "autoload": {
         "psr-4": { "Delighted\\": "lib/Delighted/" }
```
Comment (e1himself) [inline composer.json]: Please add branch alias for `dev-2.x` as `2.0-dev` (see "extra" section for example of master -> 1.0-dev).

**Comment by akalongman** [comment]
@e1himself Please look and feel free to close PR if everything is ok

File: composer.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.0-dev"
+            "dev-2.x": "2.0-dev"
```
Comment (e1himself) [inline composer.json]: @akalongman I thought we should leave "dev-master" section too. No?

Comment (akalongman) [inline composer.json]: Done

**Pull request closed without merging**